contents/handbook/people/spending-money.md

```diff
@@ -124,8 +124,8 @@ Individual software is down to your personal preference, and we encourage you to
 
 ### IDEs
 
-* IDEs range widely in cost. Best in class can cost up to \$700. We remain sceptical if a best in class IDE is worth several days of engineering time. However, we are happy to revisit this policy in the future.
```

Comment:
What IDEs cost $700 BTW? 🤔 Considering we don't do FPGAs or anything requiring awful tooling like that

Reply:
I believe that `IntelliJ IDEA Ultimate` is close to that price, yearly

Reply:
![image](https://user-images.githubusercontent.com/53387/114037601-6b19fb00-9881-11eb-95a0-70136aaf5e97.png)

Reply:
If you're working with a lot of different tools and need excellent debugging on all of them, the all products pack makes sense.

Reply:
Wow, interesting, I thought all the JetBrains IDEs were similarly priced. Good thing PostHog is Django and not Spring then LOL

Reply:
That's good for a lot of reasons 😁